src/containers/email/index.tsx

```diff
@@ -1,18 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import styled from 'styled-components';
-import { Typography } from 'antd';
+import { Select, Typography, Button } from 'antd';
 import { Routes } from '../../App';
 import PageLayout from '../../components/pageLayout';
 import { ReturnButton } from '../../components/themedComponents';
 import PageHeader from '../../components/pageHeader';
+import { EmailType, EmailerFilters } from './types';
+import EmailerFilterControls from '../../components/emailerFilterControls';
+import { CloseOutlined } from '@ant-design/icons';
 
 const EmailPageContainer = styled.div`
   width: 90vw;
   margin: 30px auto auto;
 `;
 
+const selectStyles = { marginTop: 20, minWidth: 150 };
+
+const NeighborhoodFilter = styled(Button)`
+  padding: 1px 8px;
+  margin: 0px 5px;
+  line-height: 0.8;
+`;
+
 const Email: React.FC = () => {
+  const [emailType, setEmailType] = useState<EmailType>(EmailType.INACTIVE);
+  const [filters, setFilters] = useState<EmailerFilters>({
+    activityCountMin: 0,
+    neighborhoods: [],
+    commonNames: [],
+  });
+
+  function generateFilterButtons(key: 'neighborhoods' | 'commonNames') {
+    return filters[key].map((n) => {
+      return (
+        <NeighborhoodFilter
```

Comment:
⛏️ Could we rename this to something like `Filter` just because it's used for both neighborhood and common name filters?